externs/olx.js

```diff
@@ -386,6 +387,17 @@ olx.MapOptions.prototype.logo;
 
 
 /**
+ * The minimum distance in pixels the cursor must move to be detected
+ * as a map move event instead of a click. Increasing this value can make it
+ * easier to click on the map.
+ * Default is `1`.
```

Comment:
> Default is window.devicePixelRatio.
Depends on how we define 'pixel'. Elsewhere, e.g. for `ol.Map#getSize`, we use JS pixels. So I'd prefer to leave this as `1`.